Services/ProductsService.js

```diff
@@ -13,7 +13,13 @@ const getSingleProduct = async (connection, sku) => {
     return response
 }
 
+const deleteSingleProduct = async(connection, sku) =>
```

Comment:
adding a single space between async and the params for continuity

Reply:
good shout